Sources/xcproj/PBXProj.swift

```diff
@@ -114,33 +114,53 @@ final public class PBXProj: Decodable {
         }
 
         public func getReference(_ reference: String) -> PBXObject? {
-            // The ?? chaining below is partitioned into separate statments because, with Swift 4, a single
-            // expression resulted in this compiler error:
+            // This if-let expression is used because the equivalent chain of `??` separated lookups causes,
+            // with Swift 4, this compiler error:
```

Comment:
I don't think this comment is needed anymore. If let'ing like this is a standard practice, so no need to explain it 👍

Reply:
The other two functions aren't using `if-let`s, so I wanted to explain why this one is different. Prior to this investigation, I wouldn't have been able to look at this code and know why it ended up being written this way. An `if-let` this large looks a bit off in my opinion.

Reply:
Another reason I thinks this comment is valuable is it would be good if this `if-let` could one day be converted to use a chain of `??` lookups. This comment provides that context.

Reply:
Fair enough 👍